misaligned.py

```diff
@@ -1,13 +1,20 @@
 
 def print_color_map():
+    color_map_output = []
     major_colors = ["White", "Red", "Black", "Yellow", "Violet"]
     minor_colors = ["Blue", "Orange", "Green", "Brown", "Slate"]
     for i, major in enumerate(major_colors):
         for j, minor in enumerate(minor_colors):
             print(f'{i * 5 + j} | {major} | {minor}')
-    return len(major_colors) * len(minor_colors)
+            color_map_output.append(f'{i * 5 + j} | {major} | {minor}')
+    #return len(major_colors) * len(minor_colors)

```

Comment:
Don't leave commented code in commits. If you need older code, it will be present in the source control...